src/platform/nrfconnect/SystemPlatformConfig.h

```diff
@@ -48,7 +48,7 @@ struct ChipDeviceEvent;
 #define CHIP_SYSTEM_CONFIG_USE_SOCKETS 1
 
 #ifndef CONFIG_ARCH_POSIX
-#define CHIP_SYSTEM_CONFIG_PACKETBUFFER_POOL_SIZE 15
+#define CHIP_SYSTEM_CONFIG_PACKETBUFFER_POOL_SIZE 11
```

Comment:
Is this selected based on the top usage in tests?

Reply:
Also, please fill in this info in the commit description :)

Reply:
As this is not yet targeted for 2.8.0 and we have a bit of time - what do you think to introduce a KConfig for this configuration?

Reply:
> As this is not yet targeted for 2.8.0 and we have a bit of time - what do you think to introduce a KConfig for this configuration?

Good idea. 


> Is this selected based on the top usage in tests?

Yes, highest observed pool usage was 10. I updated commit description.